README.md

```diff
@@ -12,13 +12,66 @@ A user-friendly IRC bouncer.
 
 ## Usage
 
-    go run ./cmd/sojuctl create-user <username> -admin
+### Server side
+
+    go run ./cmd/sojuctl create-user <soju username> -admin
     go run ./cmd/soju -listen irc+insecure://127.0.0.1:6667
 
-Then connect with username `<username>/chat.freenode.net` and join `#soju`.
+### Client side
+
+soju can operate in two different modes: multi connection and single 
```

Comment:
"Connection" is misleading here, since soju supports multiple clients connected at the same time. That's why "upstream " or "upstream server" is used in the man page.